guide/designing-products/open-design.md

```diff
@@ -51,68 +52,105 @@ Illustration sources
 
 # Open design
 
-**Open design is an alternative design and development method based on the free exchange of information.**
+**Open design welcomes the free exchange of ideas, knowledge, processes, and tools. It empowers by lifting barriers to innovation. It opens the door to scaling design projects by distributing adaptable resources.**
 
-By balancing the independence of individual designers and the collective power of collaboration, open design enables and promotes the convictions and passions of designers working towards the benefit of society.
+## A balance of independence and collaboration
 
-Open design can manifest in many different ways. You might:
+Open source projects are often driven by global communities. These hubs of learning and building allow you to carve an individual niche. Align yourself to projects based on your passions, talents, interests, and beliefs. Your work becomes a voluntary public contribution toward common goals. Some projects are also supported through grants and donations.
 
-* help design the interface of your favorite open-source project
-* collaborate with people and projects in areas related to specific topics of interest
-* create a design resource library like an icon set or a UI kit
-* help test an open-source product and provide feedback
-* [contribute](/guide/contribute/) to the Bitcoin Design Guide
+Open design communities can have multidisciplinary teams with designers, developers, copywriters, translators, project managers.... These diverse groups can develop stronger solutions through varied points of view.
```

Comment:
Do you think it's worth mentioning that sometimes people take on multiple roles? And that you don't need to be trained or have professional experience to help out?

Reply:
```suggestion
Open design communities can have multidisciplinary teams with designers, developers, copywriters, translators, project managers, etc. These diverse groups can develop stronger solutions through varied points of view.
```